README.md

```diff
@@ -1 +1,79 @@
-# jest-tcm-reporter
\ No newline at end of file
+[![Build Status](https://travis-ci.org/optimaxdev/jest-tcm-reporter.svg?branch=master)](https://travis-ci.org/optimaxdev/jest-tcm-reporter)
+# jest-tcm-reporter
+
+This package sends reports to the JTM
+
+## Installation
+
+```bash
+$ yarn add --dev optimaxdev/jest-tcm-reporter#master
+```
+
+## Configuration
+
+Create config file `jest-tcm.config.js`
+
+```js
+// jest-tcm.config.js
+
+module.exports = {
+    tcm: {
+        user: {
+            username: 'user',
+            password: 'password',
+        },
+        projectKey: 'ANT'
```

Comment:
The "projectKey" can be any, not necessarily "ANT", I think it's worth delete it and just writing KEY